### PR TITLE
fix typo and semantic error maybe.

### PR DESCRIPTION
**Pull request by nagexiucai**
should we start a independent repository for chinese translation?
i complete them today, see [here](https://github.com/nagexiucai/python-llfuse-docs-zh-cn).
it not only contains documents translated from http://www.rath.org/llfuse-docs/ but also some faq in future.

### CODE REVIEW

File: src/operations.pxi [hunk 1]
```diff
@@ -312,8 +312,8 @@ class Operations(object):
         *fh* will by an integer filehandle returned by a prior `open` or
         `create` call.
 
-        This method must returns the number of bytes written. However, uuless
-        the file system has been mounted with the ``direct_io`` option, the file
```
Comment (Nikratio) [inline src/operations.pxi]: this is actually correct

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (Nikratio) [inline .gitignore]: why? what's this?

File: src/operations.pxi (hunk 1 above)
Comment (nagexiucai) [reply src/operations.pxi]: first, i can not understand the word "uuless" :(

second, if "uuless" == "unless" , then :
"unless bob has been outdoor **with** a umbrella, bob will wet"
or
"unless bob has been outdoor **without** a umbrella, bob will wet" ?
mybe i do not understand the unless clause (or syntax in english) :(

File: .gitignore (hunk 2 above)
Comment (nagexiucai) [reply .gitignore]: sorry, i will delete it.
cause i read source codes of python-llfuse by pycharm from jetbrains, the ".idea" is its configuration directory of a project.

**Pull request merged**